frontend_v2/src/app/components/challenge/challenge-routing.module.ts

```diff
@@ -12,8 +12,8 @@ import { ChallengesubmissionsComponent } from './challengesubmissions/challenges
 import { ChallengeviewallsubmissionsComponent } from './challengeviewallsubmissions/challengeviewallsubmissions.component';
 import { ChallengeleaderboardComponent } from './challengeleaderboard/challengeleaderboard.component';
 import { ChallengesettingsComponent } from './challengesettings/challengesettings.component';
-import { ChallengemanageComponent } from './challengemanage/challengemanage.component';
 import { ChallengediscussComponent } from './challengediscuss/challengediscuss.component';
+import { ChallengeanalyticsComponent } from './challengeanalytics/challengeanalytics.component';
```

Comment:
@lunayach I am creating a the components using CLI and all the components are following the same pattern so do you think changing this component name explicitly to camelCase will be a good idea as it will then become different from other component classes.

Reply:
@Kajol-Kumari  It's recommended that component names are in camelCase. And this may have been followed from the beginning. But following this in one-off component would not make much of a difference, so you could leave it like this. If you have time later, this could be refactored then.